cmd/server/flags.go

```diff
@@ -77,6 +77,12 @@ var flags = []cli.Flag{
 		Usage:   "grpc address",
 		Value:   ":9000",
 	},
+	&cli.StringFlag{
+		EnvVars: []string{"WOODPECKER_METRICS_SERVER_ADDR"},
+		Name:    "metrics-server-addr",
+		Usage:   "metrics server address",
+		Value:   ":9001",
```

Comment:
Maybe we should disable it by default, so unaware users without firewalls don't leak any data?

Reply:
Agree, unprotected metrics server might be a nasty surprise for someone. Changed default to empty and updated documentation accordingly.